airflow/executors/local_kubernetes_executor.py

```diff
@@ -43,6 +43,8 @@ class LocalKubernetesExecutor(LoggingMixin):
 
     KUBERNETES_QUEUE = conf.get("local_kubernetes_executor", "kubernetes_queue")
 
+    is_local: bool = False
```

Comment:
This is tricky, since this executor is sometimes local, but I think it's best to go with False unless all executors are local.

Reply:
Yep I thought the same. This is the conservative choice, and equivalent to what was implemented before

Reply:
Aagree.

Reply:
Yeah i mean, the flag itself is problematic.  Because, with these composite executors, it's not yes or no.

Is it local? well, yes and no -- depends what you need, depends why you are asking.  So this flag should probably be replaced with something more specific to what it's actually concerned about.  Iike, the specific behavior that is required / investigated and not whether the executor is "a local executor".

Reply:
I thinkg better question is - what is the use and semantic of the "local" flag - why it is needed and what we want to achieve by it. I.e. what different happens when executor is local vs. non-local - and can we name it differently in this case to better reflect the meaning.

Reply:
that's basically what i'm saying. is_local doesn't make sense when applied to these executors, it's probably a proxy for some behavior that .... prior to the composite executors made sense but no longer does.